src/web/helpers/color/setElementsColor.ts

```diff
@@ -51,9 +52,8 @@ const setElementsColor = (elements: Element[], color: string, isFullColor = fals
           svgByLayer = true;
         }
         descendants.push(...(elem.childNodes as unknown as Element[]));
-        const href = $(elem).attr('href') || $(elem).attr('xlink:href');
-        const shadowRoot = $(href).toArray();
-        descendants.push(...shadowRoot);
+        const symbol = getRealSymbol(elem);
```

Comment:
This somehow break the color display for laser layer
because the color of image `<use>` is control the by the filter attribute of image in the image `<symbol>`